backend/sse/src/proto/hidden_test_process.proto

```
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package hidden_test_process;
```

Buf (1.47.2)

Files with package "hidden_test_process" must be within a directory "hidden_test_process" relative to root but were in directory "backend/sse/src/proto".

(PACKAGE_DIRECTORY_MATCH)